### PR TITLE
Array_contains process

### PR DESCRIPTION
**Pull request by zansinergise**
https://git.sinergise.com/team-6/openeo-platform/-/issues/63

### CODE REVIEW

File: tests/unit_tests/test_array_contains.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return load_process_code("array_contains")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({'data': [1,2,3], 'value': 2}, True),
+        ({'data': ["A","B","C"], 'value': "b"}, False),
+        ({'data': [1,2,3], 'value': "2"}, False),
+        ({'data': [1,2,None], 'value': None}, False),
```
Comment (dgostencnik) [inline tests/unit_tests/test_array_contains.py:19]: This is not in line with [Example 4](https://docs.openeo.cloud/processes/#array_contains), however it seems that either example 4 or operation definition is wrong (last bullet).

**Comment by dgostencnik** [comment]
additional cases that should be handled (red asterisk marks mandatory argument) 
 - data is undefined ({"value": "2"}, raise_ex),
 - data is null ({"data": None, "value": "2"}, raise_ex),
 - value is undefined ({"data": [1, 2, 3]}, raise_ex),

File: tests/unit_tests/test_array_contains.py (hunk 1 above)
Comment (zansinergise) [reply tests/unit_tests/test_array_contains.py:19]: I went with the assumption that the definition is correct

**Comment by zansinergise** [comment]
Fixed the process accordingly and added these test cases

**Comment by zansinergise** [comment]
> additional cases that should be handled (red asterisk marks mandatory argument)
> 
> * data is undefined ({"value": "2"}, raise_ex),
> * data is null ({"data": None, "value": "2"}, raise_ex),
> * value is undefined ({"data": [1, 2, 3]}, raise_ex),

Fixed the process accordingly and added these test cases

**Review by dgostencnik — APPROVED** [review]

**Pull request merged**